### ISSUE TITLE
Feature request: An option to auto merge "feat tracks" album

### ISSUE THREAD

**Issue by Magikh42** [krateng/maloja#335]
Hello I have a problem when feat tracks with multiple artists generate a new album

Exemple (I have fixed it in this screenshot):
![image](https://github.com/krateng/maloja/assets/44654462/bf94a73e-0d87-4394-83f7-9d33f949274d)
"`Pendulum, In Flames – Self vs Self`" was considered another album (name: Immersion ; artsits: Pendulum, In Flames), I had to merge it to Immersion manually but I think it should automatically assign to the album if the primary (first) artist is identical

**Comment by ShadowsDieThrice** [comment]
Also having this issue unfortunately. It defeats the purpose of why I use this software, because my albums are broken up into several chunks so I don't get quick and accurate data for which albums I've listened to the most.

**Comment by SirMartin** [comment]
Yes this is a big pain in the ass, hopefully, can some extra information recovered from the scrobbler and maloja would know which is the album artist. Or maybe something like suppose if one of the artists from the song has another album with the same title, make them together.

**Comment by ShadowsDieThrice** [comment]
I have figured out something of a workaround, by changing a setting in the settings.ini like so:

Delimiters_feat = ['NULLVALUE']

This prevents songs with 'feat', 'ft', etc., counting the featured artist as an artist, as the features are included in the song title instead of being recognised separately. I reimported my data from last.fm and the albums didn't separate; however I believe if your scrobbles include multiple artists in the artist category, rather than as a feature in the title, this issue will persist.

**Comment by SirMartin** [comment]
To be honest I like how it is, and for now I am taking my most listened artists and fixing the albums one by one, I taking the occasion to listen again some old albums, and I like to see when an artist appears in other ones albums, but yes is far from perfect, hopefully @krateng finds a good solution for it
